index.js

```diff
@@ -91,21 +87,7 @@ function applyStyle() {
 	return str;
 }
 
-function init() {
-	var ret = {};
-
-	Object.keys(styles).forEach(function (name) {
-		ret[name] = {
-			get: function () {
-				return build.call(this, [name]);
-			}
-		};
-	});
-
-	return ret;
-}
-
-defineProps(Chalk.prototype, init());
```

Comment:
This is to just make the code simpler. and the logic is pretty much the same as generating the `styles` above so I'm reusing it.